### DISCUSSION TITLE
Google Play failed to upload artefacts with 401

### DISCUSSION THREAD

**Discussion by br1anchen** [codemagic-ci-cd/codemagic-docs#1505] [Q&A]
We are getting `401` error with `UNAUTHENTICATED` status when upload signed artifacts to Google Play store. And App is not in draft status.

```
Google Play failed to upload artefacts. Request is missing required authentication credential. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.: {
    "error": {
        "code": 401,
        "message": "Request is missing required authentication credential. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.",
        "status": "UNAUTHENTICATED"
    }
}
```

Firebase App Distribution is also failing with timed out.

We are following this guide setup: https://docs.codemagic.io/yaml-publishing/google-play/, which has been working for some time, we did not see any changes for our google cloud service account either.

Any idea, what could cause this? And any part of Codemagic config we can look into?

**Comment by icarusdust** [comment]
Hi, as you raised the query in Intercom, let's continue there.
